feature-libs/product/configurators/rulebased/rulebased-configurator.module.ts

```diff
@@ -14,4 +14,10 @@ import { VariantConfiguratorOccModule } from './occ/variant/variant-configurator
     CpqConfiguratorRestModule,
   ],
 })
-export class RulebasedConfiguratorModule {}
+export class RulebasedConfiguratorModule {
```

Comment:
Why this change?

Reply:
seems to be leftover from my various tries to get the interceptor working in a lazy environment - i'll revert.